sync/system.performance.yml

```diff
@@ -3,15 +3,18 @@ _core:
 cache:
   page:
     max_age: 0
+    use_internal: false
 css:
-  preprocess: true
-  gzip: true
+  preprocess: false
```

Comment:
Fix it back please here and below.

To set this settings only for local development add to your **settings.local.php** the following:
$config['system.performance']['css']['preprocess'] = FALSE;